README.md

```diff
@@ -1,3 +1,4 @@
+![webgpu-napi](https://user-images.githubusercontent.com/1641613/73613765-d35ab600-45c6-11ea-813d-252444ec16eb.png)
```

Comment:
I think we should leave a logo for now. Normally, as soon as a project of mine turns into something serious/usable, I buy a logo from an artist for a few bucks :)

Reply:
Removed awesome logo :)